src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1974,27 +1043,28 @@ bool entitiesQuery
         }
       }
     }
-    else if (sco->type == SCOPE_TYPE_SIMPLE_QUERY)
+    else if (scopeP->type == SCOPE_TYPE_SIMPLE_QUERY)
     {
-      // FIXME P4: Once issue #1705 is implemented, this check can be removed.
-      if (qStringFilters(sco->value, filters) != true)
+      for (unsigned int ix = 0; ix < scopeP->stringFilter.mongoFilters.size(); ++ix)
```

Comment:
entitiesQuery is called from a number of places. Difficult to respond exactly to this question.
But yes, if the q-string is in the payload, the parsing is done in the JSON parsing stage.
If URI param (GET /v2/entities), then the parse is done in the service routine.